float64[] joint_angles #机械臂目标关节角度
float64[] matrix #4*4手眼变换矩阵，按行主序展开16个数
string reference_frame #目标参考系名称
